cmd/db_migrations.go

```diff
@@ -55,23 +55,17 @@ LIMIT 1;
 }
 
 func upgradeDBIfNeeded(db *sql.DB) error {
-	latestVersionInDB, versionErr := fetchLatestDBVersion(db)
-	if versionErr != nil {
-		return fmt.Errorf(`Couldn't get omm's latest database version. This is a fatal error; let %s
-know about this via %s.
-
-Error: %s`,
-			author,
-			repoIssuesUrl,
-			versionErr)
+	latestVersionInDB, err := fetchLatestDBVersion(db)
+	if err != nil {
+		return fmt.Errorf("%w: %v", errCouldntFetchDBVersion, err.Error())
```

Comment:
You can use multiple %w since go 1.20


```suggestion
		return fmt.Errorf("%w: %w", errCouldntFetchDBVersion, err)
```

Reply:
You are doing it on purpose, OK